packages/map-template/vite.config.js

```diff
@@ -0,0 +1,19 @@
+import { defineConfig } from 'vite';
+import react from '@vitejs/plugin-react';
+import svgr from 'vite-plugin-svgr';
+import eslint from 'vite-plugin-eslint';
+
+export default defineConfig(() => {
+    return {
+        build: {
+            outDir: 'build'
+        },
+        plugins: [
+           react(),
+           svgr({
+            svgrOptions: {}
```

Comment:
The `svgrOptions` are left empty on purpose?

Reply:
It wasn't needed. Removed.